### ISSUE TITLE
Things to fix for next release

### ISSUE THREAD

**Issue by choldgraf** [ExecutableBookProject/sphinx-copybutton#50]
- [x] Ensure that the [Sphinx-Copybutton version number](https://github.com/choldgraf/sphinx-copybutton/blob/master/jupyter_book/__init__.py)
   is correct, and remove the `dev0` part of the version number.
   Make a PR with the new number and merge into master.
- [x] Create a new distribution for Sphinx-Copybutton by
   [following the twine release instructions](https://twine.readthedocs.io/en/latest/#using-twine)
- [x] Confirm that the new version of Sphinx-Copybutton [is posted to pypi](https://pypi.org/project/sphinx-copybutton/)
- [x] Bump the [Sphinx-Copybutton version number](https://github.com/choldgraf/sphinx-copybutton/blob/master/jupyter_book/__init__.py) to
   the next minor (or major) release and append `dev0` to the end.
- [x] Celebrate! You've just released a new version of Sphinx-Copybutton!
Terms

**Comment by choldgraf** [comment]
@mgeier and @sandrotosi wanna check to make sure this looks good?

**Comment by mgeier** [comment]
The packages seem to work, there are just two minor cosmetic problems:

(1) When unpacking the packages (`wheel` and `sdist`), the contained files all are executable.
I have no idea why that's the case, but it doesn't happen when I'm trying other packages.
Are the files executable on your local file system?

(2) The `sdist` contains the `_build` directory, I guess that's not intentional?

I normally don't use Sphinx' `Makefile` (because it's silly, see https://github.com/sphinx-doc/sphinx/issues/5618#issuecomment-441698367) and use this instead:

    python3 setup.py build_sphinx

See also https://github.com/spatialaudio/nbsphinx/blob/master/CONTRIBUTING.rst.

This is better in many ways, and as a side effect it avoids having the build directory as part of the docs directory.

What's making things worse, is that you have `_build` in your `.gitignore`, which makes it hard to spot the error.

In summary, these are two anti-patterns that I try to avoid in my own projects: Sphinx' `Makefile`s and over-eager `.gitignore` files.

**Comment by choldgraf** [comment]
@mgeier thanks for the advice! Perhaps we can tackle these as issues for the next release. I'll change the issue title to reflect the new To-Do item

**Comment by sandrotosi** [comment]
Hey @choldgraf , you also released a new version of sphinx-copybutton? :)

**Comment by choldgraf** [comment]
well I have since Jan 6th :-) but is there a feature in master that you want in the released verison? Can try to make a release tomorrow if so

**Comment by sandrotosi** [comment]
not necessarily, but there are 45 commits to master since the latest release, probably a new one is due regardless if i need it or not :)

**Comment by choldgraf** [comment]
@sandrotosi https://github.com/ExecutableBookProject/sphinx-copybutton/releases/tag/v0.2.11 :-)
